lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -142,8 +142,7 @@ TypeSP DWARFASTParserClang::ParseTypeFromClangModule(const SymbolContext &sc,
   // If this type comes from a Clang module, recursively look in the
   // DWARF section of the .pcm file in the module cache. Clang
   // generates DWO skeleton units as breadcrumbs to find them.
-  llvm::SmallVector<CompilerContext, 4> decl_context;
-  die.GetDeclContext(decl_context);
+  std::vector<CompilerContext> decl_context = die.GetDeclContext();
```

Comment:
Why the change from `llvm::SmallVector<$Type, 4>` to `std::vector`? Does this not generally contain 4 or fewer CompilerContexts anymore?

Reply:
This was purely an aesthetic choice. I felt like hardcoding the constant 4 makes a choice for something that's hugely language-dependent in a general API, and given that this is parsing DWARF the extra malloc overhead of std::vector likely does not matter in the slightest.